app/portainer/components/endpoint-list/endpoint-item/endpoint-item-controller.js

```diff
@@ -1,12 +1,39 @@
-angular.module('portainer.app').controller('EndpointItemController', [
-  function EndpointItemController() {
-    var ctrl = this;
+angular.module('portainer.app').controller('EndpointItemController', function EndpointItemController() {
```

Comment:
We'll want to talk to keep consistency inside the project for code style. 

These cosmetic changes actually makes the review process harder and break the consistency of the app.